packages/design-system/src/elements/toniq-carousel/toniq-carousel.element.ts

```diff
@@ -39,7 +39,7 @@ export const ToniqCarousel = defineToniqElement<{
 }>()({
     tagName: 'toniq-carousel',
     cssVars: {
-        'toniq-carousel-arrow-margin': '40px',
+        'toniq-carousel-arrow-margin': '-20px',
```

Comment:
These style changes are weird, decreasing margin to negative but adding padding? I'm gonna see if there's a better way.

Reply:
I'll merge this and fix on the `main` branch.

Reply:
The added padding is for the white background gradient

Reply:
As far as the centered image thing, we used to have that padding to take the place of arrows but you removed it here: https://github.com/Toniq-Labs/toniq-labs-design-system/pull/54

Reply:
Yup, and that left padding is too much imo taking almost a size of an item having a big white space plus the no arrow left icon

Reply:
Actually looking back at the PR it wasn't a padding that being removed, its the arrow itself taking space even though its hidden since there's no item to scroll to both left or right, hence the 'padding'.